packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionElement.tsx

```diff
@@ -163,7 +167,8 @@ function isGreenAmount(transaction: BrowserWalletTransaction, accountAddress: st
         return false;
     }
     if (
-        transaction.type === AccountTransactionType.TransferToPublic &&
+        (transaction.type === AccountTransactionType.TransferToPublic ||
+            transaction.type === AccountTransactionType.UpdateSmartContractInstance) &&
```

Comment:
Could do

```suggestion
        ([AccountTransactionType.TransferToPublic, AccountTransactionType.UpdateSmartContractInstance].includes(transaction.type)) &&
```

Makes it easier to expand.